docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -28,8 +28,22 @@ components:
       pattern: '^([0-9A-Za-z_~\-\.]+[0-9A-Za-z_~\-])$'
       minLength: 1
       maxLength: 1023
-    AuthChannelsType:
-      title: AuthChannelsType
+    AuthenticationChannelType:
```

Comment:
This is part of the `Authorization` api resource, from the fspiop-api, so I would call this `AuthorizationChannelType` (ref: https://docs.mojaloop.io/mojaloop-specification/documents/API%20Definition%20v1.0.html#66-api-resource-authorizations)

Reply:
done. thanks for clarifying